src/buildercore/terraform.py

```diff
@@ -123,15 +128,17 @@ def render_fastly(context):
             backends.append(_fastly_backend(
                 backend['hostname'],
                 name=name,
-                request_condition=backend_condition_name
+                request_condition=backend_condition_name,
+                shield=backend.get('shield')
```

Comment:
Doesn't respect `context['fastly']['shielding']`. Should it?

Reply:
Can see a use case where the `shield` configuration is fixed for all environments or projects, but `shielding` is turned on only for a few; so yes, should be stripped if `shielding` is `False`

Reply:
(the two cases "explicit-backends" and "default `full_hostname`" will also be consistent with each other then)